### PR TITLE
Kit 28

### PR DESCRIPTION
**Pull request by emmirose**

### CODE REVIEW

File: src/app/components/collection/collection.component.scss
Comment (MatteiT) [inline src/app/components/collection/collection.component.scss:1]: ::host ? and more comments ?

Comment (emmirose) [reply src/app/components/collection/collection.component.scss:1]: All good

**Review by MatteiT — DISMISSED** [review]

**Review by MatteiT — DISMISSED** [review]

**Review by MatteiT — APPROVED** [review]

File: src/app/components/collection/collection.component.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
+@use "/src/imports" as *;
```
Comment (Benja-san) [inline src/app/components/collection/collection.component.scss:1]: please wrap your targets for better scss application

File: src/app/components/dash-collections/dash-collections.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<p>dash-collections works!</p>
```
Comment (Benja-san) [inline src/app/components/dash-collections/dash-collections.component.html:1]: this tag is not needed anymore

File: src/app/pages/collections/collections.component.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+@Component({
+  selector: 'app-collections',
+  templateUrl: './collections.component.html',
+  styleUrls: ['./collections.component.scss'],
+})
+export class CollectionsComponent {
+  collections: Collection[] = [];
+  collection!: Collection;
+  title = new FormControl('');
```
Comment (Benja-san) [inline src/app/pages/collections/collections.component.ts:14]: For future reference, can be refactored as a form group with formBuilder

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/components/collection/collection.component.scss (hunk 1 above)
Comment (emmirose) [reply src/app/components/collection/collection.component.scss:1]: Done !

File: src/app/components/dash-collections/dash-collections.component.html (hunk 2 above)
Comment (emmirose) [reply src/app/components/dash-collections/dash-collections.component.html]: Done

File: src/app/pages/collections/collections.component.ts (hunk 3 above)
Comment (emmirose) [reply src/app/pages/collections/collections.component.ts:14]: Will be done in Sprint 2 !

**Review by MatteiT — APPROVED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**